vispy/color/colormap.py

```diff
@@ -1085,48 +1085,56 @@ def __init__(self, limits=(0.33, 0.66, 1.0)):
     GrBu_d=_Diverging(255, 133, 0.75, 0.6, "dark"),
     RdBu=_Diverging(220, 20, 0.75, 0.5),
 
-    # Configurable colormaps
-    cubehelix=CubeHelixColormap,
-    single_hue=_SingleHue,
-    hsl=_HSL,
-    husl=_HUSL,
-    hsluv=_HSLuv,
-    diverging=_Diverging,
-    RdYeBuCy=_RedYellowBlueCyan,
+    cubehelix=CubeHelixColormap(),
+    single_hue=_SingleHue(),
+    hsl=_HSL(),
+    husl=_HSLuv(),
+    hsluv=_HSLuv(),
+    diverging=_Diverging(),
+    RdYeBuCy=_RedYellowBlueCyan(),
 )
 
 
-def get_colormap(name, *args, **kwargs):
-    """Obtain a colormap
-    Some colormaps can have additional configuration parameters. Refer to
-    their corresponding documentation for more information.
+def get_colormap(name):
```

Comment:
Just wondering whether we should have args and kwargs here and raise a DeprecationError or something.

I have no clue how many users would actually use it, but it if anyone would, it would be hard for them to see how they must now adjust their code.

Reply:
Yeah I can see what "feels" best. I think I'm going to go ahead and remove the instances of the classes that I created from the classes that were in the dictionary. I'll see how much information I can add to the versionchanged section in the get_colormap docstring.